### ISSUE TITLE
 [Accessibility] A11y_Manage.IIS.Net_Website_Screen Reader: After web sites link NVDA is reading some invisible content(Clickable) and after clicking on it one drop down is opening

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#180]
Environment Details:
 URL: https://manage.iis.net/settings/servers
 Browser:Chrome (68.0.3440.106) 
 Screen Reader
 MAS ID Violated: MAS 1.3.2

 Repro Steps:
 1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
 2.Download and install "Microsoft IIS Administration" application.
 3.Switch on the NVDA
 4.Tab till Microsoft IIS on top navigation bar and press enter
 5.Tab till website and press enter
 6.Select any name from the Grid and press enter 
 6.Tab till after the web site link on top navigation
 7.Check after web sites link NVDA is reading clickable or not and after clicking on it one drop down is opening

 Actual Result:
 After web sites link NVDA is reading some invisible content(Clickable) and after clicking on it one drop   down is opening

 Expected Result:
 Screen Reader should not read any kind of invisible content

 User Impact:
 Screen Reader users will get confused if screen Reader will read some invisible content

 Mas Reference:
 https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

Reference video:
[Bug_180_Nvda is redaing some invisible content.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335020/Bug_180_Nvda.is.redaing.some.invisible.content.pptx)

**Comment by jhkimnew** [comment]
This issue is not reproducible.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.2; #C+E-H; #.NETWebsites; #Chrome; #NVDA

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
